src/components/SelectionList/BaseSelectionList.js

```diff
@@ -242,16 +242,20 @@ function BaseSelectionList({
     };
 
     const renderItem = ({item, index, section}) => {
+        const normalizedIndex = index + lodashGet(section, 'indexOffset', 0);
         const isDisabled = section.isDisabled;
-        const isFocused = !isDisabled && focusedIndex === index + lodashGet(section, 'indexOffset', 0);
+        const isFocused = !isDisabled && focusedIndex === normalizedIndex;
+        // We only create tooltips for the first 10 users or so since some reports have hundreds of users, causing performance to degrade.
+        const showTooltip = normalizedIndex < 10;
 
         if (canSelectMultiple) {
```

Comment:
We can rename this better instead of `canSelectMultiple`

Reply:
@Santhosh-Sellavel I disagree, I think there needs to be a prop indicating if the list is multiple selection or not, because the selected items are controlled by the page itself. Also, I think it's outside the scope of this PR

Reply:
Agree with it being outside of the scope.

When commenting on using better names, please give a suggestion if possible... for example, for me `canSelectMultiple` is fine... so I'm not sure what would be better.

Reply:
@thiagobrez 

Maybe we can improve in the 3rd phase. 

SelectionList Item types
-> canSelectMultiple - UserListItem alone is confusing.
-> else - RadioItem 

I guess we should have a custom Item Render Block, UserListItem in the selection list item type is weird. We should clean up better to avoid confusion.  Maybe we can improve in the 3rd phase. 

cc: @aldo-expensify

Reply:
@Santhosh-Sellavel Yep, in 3rd phase more lists will come in, so we will have a better idea of what's possible to do here

Reply:
That sounds like a better time to make this kind of decision to me